ui/apps/platform/cypress/constants/DashboardPage.js

```diff
@@ -1,36 +1 @@
-import { severityColorMap } from '../../src/constants/severityColors';
-import scopeSelectors from '../helpers/scopeSelectors';
-import navigationSelectors from '../selectors/navigation';
-
 export const url = '/main/dashboard';
```

Comment:
This is slightly awkward being the only thing left in the file, but there didn't seem to be a more appropriate place to put it.